src/init/cli/YargsCliExtractor.ts

```diff
@@ -71,6 +71,18 @@ export class YargsCliExtractor extends CliExtractor {
    */
   private createYArgv(argv: readonly string[]): Argv {
     let yArgv = yargs(argv.slice(2));
+
+    // Error and show help message when multiple values were provided
+    // for a non Array type parameter
+    yArgv.check((args): any => {
+      for (const [ name, options ] of Object.entries(this.yargsArgOptions)) {
+        if (options.type !== 'array' && Array.isArray(args[name])) {
+          return `Multiple values for --${name} (-${options.alias}) were provided where only one is allowed`;
```

Comment:
```suggestion
    yArgv.check((args): boolean => {
      for (const [ name, options ] of Object.entries(this.yargsArgOptions)) {
        if (options.type !== 'array' && Array.isArray(args[name])) {
          throw new Error(`Multiple values for --${name} (-${options.alias}) were provided where only one is allowed`);
```

Would throw an error so we can clearly see this is an error situation. Also allows to just put the return type to `boolean` (as in general we never want to use `any`).

Reply:
Throwing an error actually ended up presenting a cleaner error message to the user!